timeless/restaurants/table_shapes/views.py

```diff
@@ -16,6 +18,16 @@
 BP = Blueprint("table_shape", __name__, url_prefix="/table_shapes")
 
 
+@pytest.mark.skip(reason="Waiting for TableShape implementation")
+class List(views.ListView):
+    """ List the TableShape """
+    model = models.TableShape
+    template_name = "restaurants/table_shapes/list.html",
+
+
+"""List.register(BP, "/")"""
```

Comment:
@marceloamadeu which implementation do we wait for exactly? I see the list() procedure was already activated anyway.